libs/form/README.md

```diff
@@ -0,0 +1,11 @@
+# form
```

Comment:
👍🏼

Reply:
1. aanpassingen doorgevoerd
2. artifacts.stories uitgebreid en aan-de-slag.stories aangepast
3. repositories, aan de slag en changelog: links aangepast
alle links met /story/ in heb ik aangepast, die waren allemaal fout - sinds upgrade